tls/s2n_signature_scheme.c

```diff
@@ -260,6 +260,28 @@ const struct s2n_signature_scheme* const s2n_sig_scheme_pref_list_20200207[] = {
         &s2n_ecdsa_sha1,
 };
 
+/* Only supports TLS1.2 */
+const struct s2n_signature_scheme* const s2n_sig_scheme_pref_list_default_fips[] = {
```

Comment:
Is this only for the signatures in the protocol or does it also influence the signatures used over the certificates? As I read the standard (see table 3-1), certificates must use at least SHA-256.

Reply:
that looks correct; updated to use min sha25 https://github.com/aws/s2n-tls/pull/3357/commits/e37b2408e8fa38bd181dfcc4f82aedc0201a2cec